### PR TITLE
Feat: Navigation added

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

**Review by Shah-Arsalan — APPROVED** [review]

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.nav-link-container{
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.header-link a{
+    text-decoration: none;
+    color: var(--text-secondary-color);
+    font-size: 1.2rem;
```
Comment (anuj-kumary) [inline components/navigation/navigation.css]: One suggestion Rohan try to use the same unit either PX, REM, or EM.

Comment (Rohanmond) [reply components/navigation/navigation.css]: Thanks..It is fixed now

**Review by anuj-kumary — APPROVED** [review]
Looks good to merge!!

**Review by ravikumar1002 — APPROVED** [review]

**Review by PeeyushPrashant — APPROVED** [review]
looks good to merge

**Review by suryaraj1 — APPROVED** [review]

**Review by Sharmaa-Ansh — APPROVED** [review]

**Pull request merged**